learning_resources_search/tasks_test.py

```diff
@@ -133,6 +135,9 @@ def test_start_recreate_index(mocker, mocked_celery, user, indexes):
             key=lambda course: course.learning_resource_id,
         )
 
+        for course in ocw_courses:
+            ContentFileFactory.create(run=course.learning_resource.runs.first())
```

Comment:
minor nitpick, feel free to disregard, but it might be valuable to create multiple content files per course,  maybe >   `settings.OPENSEARCHOPENSEARCH_DOCUMENT_INDEXING_CHUNK_SIZE` so that chunking can be tested.